src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -52,34 +44,42 @@ public Command parseCommand(String userInput) throws ParseException {
         logger.fine("Command word: " + commandWord + "; Arguments: " + arguments);
 
         switch (commandWord) {
```

Comment:
the indentation of the switch case statements does not adhere to coding standards